docs/dev.md

```diff
@@ -95,30 +95,27 @@ Note: Open Liberty images with the `kernel-slim` tag are not supported with dev
 
-Dev mode modifies the Dockerfile in order to allow for hot deployment. It only supports the application under development in the current project so it relies on the `.war.xml` file generated by the application build operation. This file is functionally equivalent to the `.war` file but it references class and resource files to pick up changes. To avoid application conflicts, dev mode removes all Dockerfile commands that copy or add a `.war` file. 
+Dev mode modifies the Dockerfile in order to allow for hot deployment. Hot deployment is possible because the application is installed as a loose application WAR. This method uses a file type of `.war.xml` which is functionally equivalent to the `.war` file. Dev mode only supports the application under development in the current project so to avoid application conflicts, dev mode removes all Dockerfile commands that copy or add a `.war` file.
 
-The `.war.xml` file is generated in the `defaultServer/apps` or the `defaultServer/dropins` directory so these directories are mounted in the container. Therefore any files that the Dockerfile may have copied into these directories in the container will not be accessible.
+The `.war.xml` file is generated in the `defaultServer/apps` or the `defaultServer/dropins` directory so these directories are mounted in the container. Therefore any files that the Dockerfile may have copied into these directories in the container image will not be accessible.
 
-There are other features of the Dockerfile which are not supported for hot deployment of changes. If you use any of these features to COPY or ADD files you must manually restart the container after changing the files.
-- files identified using variables in the COPY or ADD commands (e.g. `$LOCATION/file.txt`)
-- files copied from a different part of a multistage Docker build (e.g. use of the `--from` option)
-- files identified by a regular expression (e.g. `resource*.txt`)
+There are other features of the Dockerfile which are not supported for hot deployment of changes. See the section on [File Tracking](#File-Tracking) for details.
 
-Finally, if dev mode detects the Liberty command `RUN configure.sh` it will insert the environment variable command `ENV OPENJ9_SCC=false` in order to skip the configuration of the shared class cache.
+Finally, if dev mode detects the Liberty command `RUN configure.sh` it will insert the environment variable command `ENV OPENJ9_SCC=false` in order to skip the configuration of the shared class cache. For more information see [OpenJ9 Shared Class Cache](https://github.com/OpenLiberty/ci.docker/#openj9-shared-class-cache-scc).
```

Comment:
Could remove the last sentence by just hyperlinking "shared class cache" in the previous sentence (so reduce overall lengthiness).

Reply:
Ok